ometiff-JavaScript/PixelSizeConsistency.js

```diff
@@ -47,12 +47,18 @@ getExif().then(function(result) {
     
     parseString(xml, function (err, output) {
 
-        const physicalSizeX = output['OME']['Image'][0]['Pixels'][0]['$']['PhysicalSizeX']
-        const physicalSizeXUnit = output['OME']['Image'][0]['Pixels'][0]['$']['PhysicalSizeXUnit']
-        const physicalSizeY = output['OME']['Image'][0]['Pixels'][0]['$']['PhysicalSizeY']
-        const physicalSizeYUnit = output['OME']['Image'][0]['Pixels'][0]['$']['PhysicalSizeYUnit']
-        const physicalSizeZ = output['OME']['Image'][0]['Pixels'][0]['$']['PhysicalSizeZ']
-        const physicalSizeZUnit = output['OME']['Image'][0]['Pixels'][0]['$']['PhysicalSizeZUnit']
+        let rootKey = Object.keys(output)[0]
+        let namespace = ''
+        if (rootKey.includes(':')) {
+          namespace = rootKey.split(':')[0].concat(':')
```

Comment:
```suggestion
        if (rootKey.includes(':OME')) {
          namespace = rootKey.split(':OME')[0].concat(':')
```
I don't know if that makes sense, but just to be sure we are dealing with and "OME" rootkey.
(and I don't know if `<anything>:<anything>:OME` could exist?)